Helper/Data.php

```diff
@@ -238,19 +238,44 @@ public function checkDeliveryType(?int $deliveryType): int
     }
 
     /**
-     * @param  int    $order_id
+     * @param  int    $orderId
      * @param  string $status
      */
-    public function setOrderStatus(int $order_id, string $status): void
+    public function setOrderStatus(int $orderId, string $status): void
     {
         $order = ObjectManager::getInstance()
             ->create('\Magento\Sales\Model\Order')
-            ->load($order_id);
+            ->load($orderId);
         $order->setState($status)
             ->setStatus($status);
         $order->save();
+    }
+
+    public function consignmentHasShipmentOption(AbstractConsignment $consignment, string $shipmentOption): bool
+    {
+        /**
+         * Business logic determining what shipment options to show, if any.
+         */
+        if (! $consignment->canHaveShipmentOption($shipmentOption)) {
+            return false;
+        }
+
+        if (AbstractConsignment::CC_BE === $consignment->getCountry()) {
+            if (CarrierPostNL::NAME === $consignment->getCarrierName()) {
+                if (!in_array($shipmentOption, [
+                    AbstractConsignment::SHIPMENT_OPTION_ONLY_RECIPIENT,
+                    AbstractConsignment::SHIPMENT_OPTION_SIGNATURE], true)
+                ) {
+                    return false;
+                }
+            } else {
+                return false;
+            }
+        } elseif (AbstractConsignment::CC_NL !== $consignment->getCountry()) {
+            return false;
+        }
```

Comment:
Ik denk dat we deze met een kleine switcharoo in logica nog wat kleiner kunnen krijgen, als ik de business logic goed heb begrepen, dus controleer dit misschien ook even :)

```suggestion
        // Netherlands ALWAYS has shipment options
        if (AbstractConsignment::CC_NL === $consignment->getCountry()) {
            return true;
        }
        // For PostNL in Belgium - only have shipment options for recipient-only/signature consignments
        if (AbstractConsignment::CC_BE === $consignment->getCountry() && CarrierPostNL::NAME === $consignment->getCarrierName()) {
                return in_array($shipmentOption, [
                    AbstractConsignment::SHIPMENT_OPTION_ONLY_RECIPIENT,
                    AbstractConsignment::SHIPMENT_OPTION_SIGNATURE], true);
        }
        // No shipment options available in any other cases 
        return false;
```